src/Errors/AbstractError.ts

```diff
@@ -5,6 +5,7 @@ abstract class AbstractError extends Error {
 
 	constructor( message:string ) {
 		super( message );
+		Error.captureStackTrace( this, this.constructor );
```

Comment:
You are right, this function is only available en V8. But the stack trace is not constructing correctly in Node, so I will add a verification.